code/modules/mob/living/simple_animal/hostile/ordeal/misc.dm

```diff
@@ -0,0 +1,109 @@
+//If you see this, hush hush. - Kirie
+/mob/living/simple_animal/hostile/ordeal/black_dusk
+	name = "Black Fixer"
+	desc = "A humanoid creature wrapped in bandages."
+	icon = 'ModularTegustation/Teguicons/32x64.dmi'
+	icon_state = "fixer_b"
+	icon_living = "fixer_b"
+	faction = list("black_ordeal")
+	maxHealth = 4000
+	health = 4000
+	melee_damage_type = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	rapid_melee = 2
+	melee_damage_lower = 35
+	melee_damage_upper = 35
+	ranged = TRUE
+	attack_verb_continuous = "bashes"
+	attack_verb_simple = "bashes"
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.7, WHITE_DAMAGE = 0.4, BLACK_DAMAGE = 0.0, PALE_DAMAGE = 0.4)
+	move_resist = MOVE_FORCE_OVERPOWERING
+	projectiletype = /obj/projectile/black
+	var/pulse_cooldown
+	var/pulse_cooldown_time = 15 SECONDS
+	var/pulse_damage = 30
+	var/busy = FALSE
+
+/mob/living/simple_animal/hostile/ordeal/black_dusk/Life()
+	. = ..()
+	if(!.) // Dead
+		return FALSE
+	if(pulse_cooldown < world.time)
+		pulse()
+
+/obj/projectile/black
+	name = "kunai"
+	icon_state = "blackfixer"
+	knockdown = 5
+	damage = 30
+	damage_type = BLACK_DAMAGE
+
+/mob/living/simple_animal/hostile/ordeal/black_dusk/proc/pulse()
+	icon_state = "fixer_b_attack"
+	SLEEP_CHECK_DEATH(5)
+	pulse_cooldown = world.time + pulse_cooldown_time
+	playsound(src, 'sound/weapons/resonator_blast.ogg', 100, FALSE, 90)
```

Comment:
> 90

Yeah, no, that is too much. If you want everyone affected to hear it just make it a directed sound below.

Reply:
Additionally, this will instantly proc on spawn, causing an inevitable breach or two. Or three... Or ten...
Set cooldown on Initialize, see how it's done for Claw.

Reply:
Swapped it to the same range of the attack, and resolved.